<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="features_number_spin_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>100</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="display_homography_check_box">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Display homography</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="simple_match_distance_coef_double_spin_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="knn_match_distance_ratio_double_spin_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>0.990000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_knn_match_check_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use KNN matching</string>
   </property>
  </widget>
  <widget class="QLabel" name="features_number_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of features</string>
   </property>
  </widget>
  <widget class="QLabel" name="knn_match_distance_ratio_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>KNN match distance ratio</string>
   </property>
  </widget>
  <widget class="QLabel" name="simple_match_distance_coef_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple match distance coef</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_ransac_filterring_check_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use RANSAC filterring</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ransac_distance_double_spin_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="ransac_distance_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RANSAC distance</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ransac_confidence_double_spin_box">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="ransac_confidence_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RANSAC confidence</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ransac_refine_fundamental_matrix_check_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RANSAC refine fundamental matrix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="resize_input_image_check_box">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize input image</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_image_max_dimmension_size_spin_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>300</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="input_image_max_dimmension_size_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input image max dimenssion size</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>21</width>
     <height>231</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="crop_image_check_box">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop image</string>
   </property>
  </widget>
  <widget class="QLabel" name="crop_padding_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop padding</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="crop_padding_spin_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>45</number>
   </property>
  </widget>
  <widget class="QComboBox" name="grayscale_resize_mode_combo_box">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string comment="0" extracomment="Does not resize the image.">Original</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="1" extracomment="Resizes the image to half its size.">Shrink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string comment="2" extracomment="This will shrink the image to half its size and then will expand it back to full size.">Blur by resizing</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
